core/internal/features/features_test.go

```diff
@@ -1354,6 +1355,12 @@ func TestIntegration_BlockHistoryEstimator(t *testing.T) {
 		require.NoError(t, err)
 		return gasPrice.Legacy.String()
 	}, testutils.WaitTimeout(t), cltest.DBPollingInterval).Should(gomega.Equal("45 gwei"))
+
+	defer func() {
+		for _, re := range cc.Slice() {
+			require.NoError(t, re.Close())
+		}
+	}()
```

Comment:
The `Close` calls should be set up just after `Start`, on 1322. But there is actually a helper func you can use to make this eaiser:
```go
for _, re := range cc.Slice() {
	servicetest.Run(t, re)
}
```
It will start them, as well as register a cleanup func to close them, and it uses `assert` so that every close is always called.

Reply:
Oh yeah, I used this extensively in a recent PR but forgot about it 👍